pkg/config/analysis/analyzers/deprecation/deprecation.go

```diff
@@ -114,6 +121,15 @@ func (*FieldAnalyzer) analyzeSidecar(r *resource.Instance, ctx analysis.Context)
 	}
 }
 
+func (*FieldAnalyzer) analyzeProxyConfig(r *resource.Instance, ctx analysis.Context) {
+	pc := r.Message.(*v1alpha1.ProxyConfig)
```

Comment:
This value can be either in ProxyConfig or MeshConfig right?





[5:32](https://istio.slack.com/archives/D05JU9R3V5K/p1736958763192279)
so the check you have added works for both?